### PR TITLE
Cypress/fe 1480

### PR DESCRIPTION
**Pull request by marcinstachowicz**
## Automation tests for Storybook components

**Improvements:**
- locator files structure

### CODE REVIEW

File: cypress/locators/alert-dialog/index.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
+import {
+  SHOW_CLOSE_ICON_CHECKBOX, STICKY_FORM_FOOTER, DISABLE_ESC_KEY_CHECHBOX, CLOSE_ICON_BUTTON,
```
Comment (DlgSHi) [inline cypress/locators/alert-dialog/index.js]: typo DISABLE_ESC_KEY_CHECHBOX -> checKbox

File: cypress/locators/checkbox/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-const FIELD_HELP_INLINE_CHECKBOX = '#fieldHelpInline';
 export const reverseCheckbox = () => cy.get(REVERSE_CHECKBOX);
 export const fieldHelpInlineCheckbox = () => cy.get(FIELD_HELP_INLINE_CHECKBOX);
 
 // component preview locators
-const CHECKBOX_HELP = '.carbon-checkbox__help-text';
-const CHECKBOX_COMMOM_INPUT_FIELD = '.carbon-checkbox > .common-input__field';
-const CHECKBOX_LABEL = '.common-input__label';
 export const checkboxHelpTextPreview = () => cy.iFrame(CHECKBOX_HELP);
 export const checkboxCommonInputField = () => cy.iFrame(CHECKBOX_COMMOM_INPUT_FIELD);
```
Comment (DlgSHi) [inline cypress/locators/checkbox/index.js]: typo in selector CHECKBOX_COMMOM_INPUT_FIELD

File: cypress/locators/confirm/locators.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// knobs locators
+export const OPEN_BUTTON = 'button[type="button"][name="open"]';
+export const CHILDREN_TEXTAREA = 'textarea[id="children"]';
+export const TITLE_INPUT = '#title';
+export const DISABLE_ESC_KEY_CHECHBOX = '#disableEscKey';
```
Comment (DlgSHi) [inline cypress/locators/confirm/locators.js]: typo

File: cypress/locators/index.js [hunk 4]
```diff
@@ -27,14 +21,10 @@ export const labelHelpInput = () => cy.get(LABEL_HELP_INPUT);
 export const inputWidthSlider = () => cy.get(INPUT_WIDTH_SLIDER);
 export const labelWidthSlider = () => cy.get(LABEL_WIDTH_SLIDER);
 export const fieldHelpInput = () => cy.get(FIELD_HELP_INPUT);
-export const labelInlineCheckbox = () => cy.get(LABLEL_INLINE_CHECKBOX);
+export const lableInlineCheckbox = () => cy.get(LABLEL_INLINE_CHECKBOX);
```
Comment (DlgSHi) [inline cypress/locators/index.js]: typo, LABLEL_INLINE_CHECKBOX

File: cypress/support/step-definitions/button-toggle-group-steps.js [hunk 5]
```diff
@@ -21,7 +21,7 @@ Then('Label help on preview is set to {string}', (text) => {
 });
 
 When('I uncheck label inline checkbox', () => {
-  labelInlineCheckbox().uncheck();
+  lableInlineCheckbox().uncheck();
```
Comment (DlgSHi) [inline cypress/support/step-definitions/button-toggle-group-steps.js]: typo ?

File: cypress/locators/configurable-items/index.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
 export const draggableItemByText = text => cy.get('.common-input__label').contains(text).parent().parent()
   .find('.icon-drag_vertical');
-export const draggableItemByPosition = position => cy.get('.carbon-configurable-items__items-wrapper').find(`li:nth-child(${position})`).find('label');
+export const draggableItemByPosition = position => cy.get('.carbon-configurable-items__items-wrapper')
```
Comment (DlgSHi) [inline cypress/locators/configurable-items/index.js]: maybe we should make the locators in locators.js file ?
the same question is for draggable-context

File: cypress/locators/alert-dialog/index.js (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/locators/alert-dialog/index.js]: fixed

File: cypress/locators/checkbox/index.js (hunk 2 above)
Comment (marcinstachowicz) [reply cypress/locators/checkbox/index.js]: fixed

File: cypress/locators/confirm/locators.js (hunk 3 above)
Comment (marcinstachowicz) [reply cypress/locators/confirm/locators.js]: fixed

File: cypress/support/step-definitions/button-toggle-group-steps.js (hunk 5 above)
Comment (marcinstachowicz) [reply cypress/support/step-definitions/button-toggle-group-steps.js]: fixed

File: cypress/locators/configurable-items/index.js (hunk 6 above)
Comment (marcinstachowicz) [reply cypress/locators/configurable-items/index.js]: agreed, done

**Pull request merged**